lib/Exporter/Writer/CsvWriter.php

```diff
@@ -29,21 +29,25 @@ class CsvWriter implements WriterInterface
 
     protected $position;
 
+    protected $hasBom = null;
```

Comment:
Should not have default value as it's set on constructor.

BTW, default values are different (`null`/`false`).